OnlineGameStore.Api/Controllers/CommentConroller.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OnlineGameStore.Application.Models.Requests;
+using OnlineGameStore.Application.Services.Interfaces;
+using OnlineGameStore.Infrastructure.Entities;
+
+namespace OnlineGameStore.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CommentConroller : ControllerBase
+    {
+
```

Comment:
Remove empty line or add it to all controllers to maintain consistency. First variant is better